plugins/grafana-custom-plugins/grafana-http-table-plugin/src/types.ts

```diff
@@ -0,0 +1,7 @@
+type SeriesSize = 'sm' | 'md' | 'lg';
```

Comment:
Are these options being used anywhere? I see they are passed as parameter into `TablePanel`, but I don't see where users can configure them, and where do we explicitly handle their values in the source code? The name "SimpleOptions" also looks a bit too general for me to understand its usage.

Reply:
those are default options, I'll remove them